src/features/analytical-env.feature

```diff
@@ -434,3 +434,113 @@ Feature: Creating Analytical Environment for End Users
     Then The user is unable to write to the location
 
+  @fixture.setup.rbac_uc_clive__non_pii_user_and_role
+  Scenario: A Non SC Clive user attempts to access non PII clive DB data in the published S3 bucket
+    Given A user is cleared to read non PII clive DB data in the published S3 bucket
+    When The user Attempts to read data tagged with the pii:false tag from the Clive database in the published S3 bucket
+    Then The user is able to read the data
+
+  @fixture.setup.rbac_uc_clive__non_pii_user
+  Scenario: A Non SC Clive user attempts to access PII clive DB data in the published S3 bucket
+    Given A user is not cleared to read PII clive DB data in the published S3 bucket
+    When The user Attempts to read data tagged with the pii:true tag from the Clive database in the published S3 bucket
+    Then The user is unable to read the data
+
+  @fixture.setup.rbac_uc_clive__non_pii_user
+  Scenario: A Non SC Clive user attempts to access PII data from the UC DB in the published S3 bucket
+    Given A user is not cleared to access PII data from the UC database in the published S3 bucket
+    When The user Attempts to read data tagged with the pii:true tag in the published S3 bucket
+    Then The user is unable to read the data
+
+  @fixture.setup.rbac_uc_clive__non_pii_user
+  Scenario: A Non SC Clive user attempts to access Non PII data from the UC DB in the published S3 bucket
+    Given A user is not cleared to access non PII data from the UC database in the published S3 bucket
+    When The user Attempts to read data tagged with the pii:false tag in the published S3 bucket
+    Then The user is unable to read the data
+
+  @fixture.setup.rbac_uc_clive__non_pii_user
+  Scenario: A Non SC Clive user attempts to access data from the team directories in the published S3 bucket
+    Given A user is not cleared to access any data in any team databases in the published S3 bucket
+    When The user attempts to read data from each of the team directories in the published S3 bucket
+    Then The user is unable to read any of the data
+
+  @fixture.setup.rbac_uc_clive__non_pii_user
+  Scenario: A Non SC Clive user attempts to access non PII data from the team directories in the published S3 bucket
+    Given A user is not cleared to access any non PII data in any team databases in the published S3 bucket
+    When The user attempts to read data from each of the team directories in the published S3 bucket
+    Then The user is unable to read any of the data
+
+  @fixture.setup.rbac_uc_clive__non_pii_user
+  Scenario: A Non SC Clive user attempts to access data in the restricted directories in the published S3 bucket
+    Given A user is not cleared to access data in the restricted directories in the published S3 bucket
+    When The user Attempts to read data from each of the restricted directories in the published S3 bucket
+    Then The user is unable to read any of the data
+
+  @fixture.setup.rbac_uc_clive__non_pii_user
```

Comment:
Why are we testing against other databases? This will grown stupidly large soon, if we are testing that n databases cannot read (n - 1) databases with r roles, it would require writing a large set of scenarios. eg. 10 databases would require us writing 180 scenarios! And onboarding a new database would require writing 40 additional tests!

Reply:
I agree - however thought it best to try follow the pattern already there, as the tests for not being able to access other databases already exist and are being used. Would it be possible to test the negation of each policy somehow instead of testing every other scenario? That would presumably have to be a separate ticket to look at all the tests again?

Reply:
I don't think we need to test every database, ultimately if the process works, then it works. We should then have something that tests the specific components that it gets put in the right db. I think testing all the policies once is fine. Maybe have all the databases tested one time for a policy where they can read data and one time where they can't and then use UC database for all other policies? Seems a reasonable balance of risk vs work.

Reply:
I think that makes most sense Steve, should I raise a new ticket to look at minimising the amount of repeated tests or cover it in this PR?